src/pages-and-resources/discussions/data/api.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/prefer-default-export */
+export function getDiscussionApps() {
+  return Promise.resolve({
+    features: [
+      {
+        id: 'lti',
+        name: 'LTI Integration',
+      },
+      {
+        id: 'discussion-page',
+        name: 'Discussion Page',
+      },
+      {
+        id: 'embedded-course-sections',
+        name: 'Embedded Course Sections',
+      },
+      {
+        id: 'embedded-course-units',
+        name: 'Embedded Course Units',
+      },
+      {
+        id: 'wcag-2.1',
+        name: 'WCAG 2.1 Support',
+      },
+    ],
+    apps: [
+      {
+        id: 'edx-forums',
+        name: 'edX Forum',
```

Comment:
random, why is the id `forums` but the name `forum`?

Reply:
It's still just test data and I hadn't yet put in effort to figure out what it should be called. 😁

Reply:
Yeah I have no idea either